sdks/python/apache_beam/dataframe/pandas_doctests_test.py

```diff
@@ -420,9 +446,9 @@ def test_top_level(self):
             'merge': [
                 "df1.merge(df2, left_on='lkey', right_on='rkey')",
                 "df1.merge(df2, left_on='lkey', right_on='rkey',\n"
-                "          suffixes=('_left', '_right'))"
-            ],
-            # Not an actual test.
+                "          suffixes=('_left', '_right'))",
+                "df1.merge(df2, how='left', on='a')",
+            ],  # Not an actual test.
```

Comment:
This comment applies to the next line, not this one.

Reply:
yapf!

Thanks, fixed